### PR TITLE
Update index.md

### PR DESCRIPTION
**Pull request by asklar**
Clarify some winui 2 details which were wrong

### CODE REVIEW

**Comment by PRMerger15** [comment]
@asklar : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: hub/apps/winui/index.md [hunk 1]
```diff
@@ -35,11 +35,11 @@ The following table highlights some of most significant differences between WinU
...
-| WinUI 3 supports C# and .NET 5 for desktop apps. | WinUI 2 supports C# and .NET Native apps only. |
-| WinUI 3 support for production-ready UWP apps is currently in preview, see [WinUI 3 - Project Reunion 0.5 Preview](winui3/release-notes/winui3-project-reunion-0.5-preview.md).                                                                                                                                | WinUI 2 can be incorporated into production UWP apps by installing a NuGet package into a new or existing UWP project. WinUI controls and styles can then be referenced directly in new apps, or by updating "windows.ui." namespace references to "microsoft.ui." in existing apps.                                                                                                                                                                                    |
-| WinUI 3 supports the Chromium-based [WebView2](/microsoft-edge/webview2/) control |  WinUI 2 supports the  [WebView](/windows/uwp/design/controls-and-patterns/web-view) control |
+| WinUI 3 supports C# .NET 5 and C++ for desktop apps. | WinUI 2 supports C# .NET Native and C++ apps. |
```
Comment (asklar) [inline hub/apps/winui/index.md]:
```suggestion
| WinUI 3 supports C# (.NET 5) and C++ for desktop apps. | WinUI 2 supports C# and Visual Basic (.NET Native), and C++ apps. |
```

**Comment by PRMerger-2** [comment]
@asklar : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: hub/apps/winui/index.md [hunk 2]
```diff
@@ -35,11 +35,11 @@ The following table highlights some of most significant differences between WinU
...
-| WinUI 3 can be used to build production-ready **desktop/Win32** Windows apps. | WinUI 2 cannot be used to build **desktop/Win32** Windows apps. |
+| WinUI 3 can be used to build production-ready **desktop/Win32** Windows apps. | WinUI 2 can be used in UWP applications and incorporated into new or existing desktop applications using [XAML Islands](../desktop/modernize/xaml-islands.md) (for installation instructions, see [Getting started with the WinUI 2 Library](winui2/getting-started.md)). |
```
Comment (riverar) [inline hub/apps/winui/index.md]: Nit: Drop "new and existing" as you're just repeating all possibilities.

```suggestion
| WinUI 3 can be used to build production-ready **desktop/Win32** Windows apps. | WinUI 2 can be used in UWP applications and incorporated into desktop applications using [XAML Islands](../desktop/modernize/xaml-islands.md) (for installation instructions, see [Getting started with the WinUI 2 Library](winui2/getting-started.md)). |
```

File: hub/apps/winui/index.md [hunk 3]
```diff
@@ -35,11 +35,11 @@ The following table highlights some of most significant differences between WinU
...
+| WinUI 3 supports C# (.NET 5) and C++ for desktop apps. | WinUI 2 supports C# and Visual Basic (.NET Native), and C++ apps. |
+| WinUI 3 support for production-ready UWP apps is currently in preview, see [WinUI 3 - Project Reunion 0.5 Preview](winui3/release-notes/winui3-project-reunion-0.5-preview.md).                                                                                                                                | WinUI 2 can be incorporated into production UWP apps by installing a NuGet package into a new or existing UWP project. WinUI controls and styles can then be referenced directly in new apps, or by updating "Windows.UI." namespace references to "Microsoft.UI." in existing apps.                                                                                                                                                                                    |
+| WinUI 3 supports the Chromium-based [WebView2](/microsoft-edge/webview2/) control |  WinUI 2 supports the  [WebView](/windows/uwp/design/controls-and-patterns/web-view) control on all devices, and the [WebView2](/microsoft-edge/webview2/) control for Desktop only. |
```
Comment (riverar) [inline hub/apps/winui/index.md]: Nit: Grammar tweak


```suggestion
| WinUI 3 supports the Chromium-based [WebView2](/microsoft-edge/webview2/) control |  WinUI 2 supports the  [WebView](/windows/uwp/design/controls-and-patterns/web-view) control on all devices, and the [WebView2](/microsoft-edge/webview2/) control on desktop. |
```

**Review by riverar — COMMENTED** [review]
Looks good, just some nits.

**Comment by ktoliver** [comment]
#label:"aq-pr-triaged"

**Comment by Karl-Bridge-Microsoft** [comment]
@gabbybilka - does this update look good to you?

**Comment by gabbybilka** [comment]
Yep, looks good to me, at the bottom of the page there is a reference to WinUI 2.7, if this PR can also update that to WinUI 2.8 linking to this [milestone](https://github.com/microsoft/microsoft-ui-xaml/milestone/14) I won't open another PR for it. Thanks!

**Comment by PRMerger8** [comment]
@asklar : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger13** [comment]
@asklar : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: hub/apps/winui/index.md (hunk 3 above)
Comment (asklar) [reply hub/apps/winui/index.md]:
```suggestion
| WinUI 3 supports the Chromium-based [WebView2](/microsoft-edge/webview2/) control |  WinUI 2 supports the  [WebView](/windows/uwp/design/controls-and-patterns/web-view) control on all devices, and starting with WinUI 2.8 prerelease, the [WebView2](/microsoft-edge/webview2/) control on Desktop. |
```

**Comment by PRMerger12** [comment]
@asklar : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Comment by PRMerger20** [comment]
@asklar : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by Karl-Bridge-Microsoft — APPROVED** [review]

**Comment by GrantMeStrength** [comment]
Closing — this issue references a UWP page (`windows-apps-src/`) that is now maintained in the [windows-uwp](https://github.com/MicrosoftDocs/windows-uwp) repository, not this one. If the issue is still relevant, please re-file it there. Thank you!

**Pull request merged**